assets/terraform-modules/packet/flatcar-linux/kubernetes/workers/variables.tf

```diff
@@ -195,3 +195,21 @@ variable "enable_tls_bootstrap" {
   description = "Enable TLS Bootstrap for Kubelet."
   type        = bool
 }
+
+variable "cpu_manager_policy" {
+  description = "CPU Manager policy to use for the worker pool. Possible values: `none`, `static`."
+  default     = "none"
+  type        = string
+}
+
+variable "kube_reserved_cpu" {
+  description = "CPU cores reserved for the Worker Kubernetes components like kube-proxy, kubelet, etc."
+  default     = "2000m"
```

Comment:
2 cores only for kubelet and etcd? That seems like a lot. How this has been estimated?

Reply:
Good point. We have this enabled only for workers not for control plane. So etcd is out of the picture. Talking about the idle kubelet yeah 2 cores can be a lot for it. I will reduce it to 500m.

Reply:
On my nodes with ~30 pods, CPU does not exceed 0.15 core. What's our max pods per node here? We could calculate as such perhaps.

Reply:
![Selection_441](https://user-images.githubusercontent.com/16539896/127285374-44557483-3651-480f-8bf1-91926eb94491.png)
CPU usage per pod per kubelet instance.

Reply:
Is `# of pods` proportional to kubelet CPU usage? Also what about the spike of CPU usage before the self-hosted takes over?

Reply:
They are limits for the kubelet when we enable the cpu-manager-policy. Because rest of the CPU is allocated to others exclusively. 

About number of CPUs I followed [this guide](https://cloud.redhat.com/blog/full-cluster-part-2-protecting-nodes) which also recommends `# of pods * 5m`. I doubled it just to be conservative.

Reply:
Do we set max number of pods or have it configurable?

Reply:
We use the defaults.

Reply:
If we use the default of 110, then I think having it at 500m or something around make sense then :+1: